components/coding/profileV2/BadgesDisplayedComponent.tsx

```diff
@@ -1,3 +1,4 @@
+
 import React from "react";
 
 import { User } from "./ProfileHeaderComponent";
```

Comment:
You moved this to fetchUserProfile so need to update the import path

Is your linter broken? I feel like it should alert you to changed imports and remove the random blank rows across your code